docs/overview/deep_dive/navigating_the_code.rst

```diff
@@ -288,19 +288,17 @@ Then, we import this submodule-specific helper function to the respective backen
         ivy.utils.assertions.check_fill_value_and_dtype_are_compatible(fill_value, dtype)
         return torch.full_like(x, fill_value, dtype=dtype, device=device)
 
-Version Pinning
+Version Unpinning
 ---------------
```

Comment:
the number of `-` should match the number of letters in the header. Just a minor change, thanks @RickSanchezStoic @KareemMAX 😄